packages/rancher-istio/charts/values.yaml

```diff
@@ -61,18 +77,15 @@ meshConfig:
 kiali:
   enabled: true
   auth:
-    # todo: what auth strategy ?
-    # strategy: token # this is default, you can use a service-account or kubeconfig token
     strategy: anonymous
   deployment:
+    ##TODO: update kiali to allow for repository and image
     ingress_enabled: false
+    image_name: docker.io/rancher/kiali-kiali
```

Comment:
Is there a reason why the Kiali image does not use the global hub value?

Reply:
I will be making all the kiali updates in a separate pr - since its a dependent chart I will have to patch the rancher-kiali-servcer chart to use the global hub value.

Reply:
Yes you'll need to add the registry helper to kiali, and that to the image spec, and it should flow through from global var here overriding kiali global var. So you wouldn't specify hub at this point, but we can leave for now.